openfe/tests/setup/test_molecule.py

```diff
@@ -1,5 +1,12 @@
+import openff.toolkit.topology
 import pytest
 
+try:
+    from openeye import oechem
+except ImportError:
```

Comment:
```suggestion
except ImportError:  # -no-cov-
```

looks like this line codecov is complaining about this; maybe we add something for the test matrix to test against a minimal set of reqs? For now, ignore coverage here.